halo2_proofs/src/plonk/evaluation.rs

```diff
@@ -434,7 +434,7 @@ impl<C: CurveAffine> Evaluator<C> {
             .enumerate()
             .take(num_clusters)
         {
-             cluster.resize(1 << cluster_idx, domain.empty_lagrange());
+            cluster.resize(1 << cluster_idx, domain.empty_lagrange());
```

Comment:
Reformat the whole thing or leave this out.

Reply:
yes, make a global cargo fmt.